shell/server/shell-server.js

```diff
@@ -60,6 +60,8 @@ BrowserPolicy.content.allowImageOrigin(staticAssetHost);
 BrowserPolicy.content.allowScriptOrigin(staticAssetHost);
 BrowserPolicy.content.allowFontOrigin(staticAssetHost);
 BrowserPolicy.content.allowConnectOrigin(staticAssetHost);
+BrowserPolicy.content.allowConnectOrigin("wss:");
```

Comment:
As part of [meteor's default policy](https://github.com/meteor/meteor/blob/cf4133c7632137d870136f02100622aa7e8fcd95/packages/browser-policy-content/browser-policy-content.js#L139), we already have `connect-src: *`, which allows websocket connects to any origin in browsers other than Safari, if I understand correctly.